### PR TITLE
Update list of assemblies required from KSP, see #2570

### PR DESCRIPTION
**Pull request by MaraRinn**
The list of assemblies has changed significantly since the last pass of this document.

This is a partial update, pending clarification on #2569 regarding possible missing assemblies.

### CODE REVIEW

**Comment by Dunbaratu** [comment]
Thank you.  I'll try to get this in before I cut the release, which is coming in a couple of days when I finish the documentation edits.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -146,12 +146,25 @@ Setting Up Your Environment
...
+		* `UnityEngine`
+		* `UnityEngine.AnimationModule`
+		* `UnityEngine.AudioModule`
+		* `UnityEngine.CoreModule`
+		* `UnityEngine.IMGUIModule`
+		* `UnityEngine.PhysicsModule`
+		* `UnityEngine.TextRenderingModule`
+		* `UnityEngine.UI`
+		* `UnityEngine.UnityWebRequestWWWModule`
+	1. If you do not have a copy of KSP locally, you may
```
Comment (Dunbaratu) [inline CONTRIBUTING.md]: 1. ?  Should this be 2?

**Comment by Dunbaratu** [comment]
This is so minor I'm just going to push the github merge button instead of doing it the "proper" way.

File: CONTRIBUTING.md (hunk 1 above)
Comment (MaraRinn) [reply CONTRIBUTING.md]: Probably, for human readers. Markdown parsers will continue the numbering so they can all be 0. or 1. or whatever. I'll fix that now.

Comment (Dunbaratu) [reply CONTRIBUTING.md]: Don't bother.  I noticed that, and already merged it.

**Pull request merged**